src/chef-mover/rebar.config

```diff
@@ -10,7 +10,7 @@
     {lager, ".*",
         {git, "git://github.com/basho/lager.git",                     {branch, "master"}}},
     {bcrypt, ".*",
-        {git, "git://github.com/smarkets/erlang-bcrypt.git",          {branch, "master"}}},
+        {git, "git://github.com/benjamin-bergia/erlang-bcrypt.git",   {branch, "master"}}},
```

Comment:
Looks like the one we used before https://github.com/smarkets/erlang-bcrypt was forked from https://github.com/abhay/erlang-bcrypt. The original upstream https://github.com/abhay/erlang-bcrypt has the changes for using strong_rand_bytes instead of rand_byte. We should use this instead of another fork.

Reply:
Will place this in a separate revertible commit for now, so we can test it.

Reply:
Builds and tests are green.